rca/programmes/models.py

```diff
@@ -607,6 +609,33 @@ class ProgrammePage(TapMixin, ContactFieldsMixin, BasePage):
         help_text="The title of the social media links section",
     )
 
+    # Experience
+    experience_introduction = models.CharField(
+        max_length=500,
+        blank=True,
+        help_text="Introductory text for the Experience section",
+    )
+    experience_cta_link = StreamField(
+        [("link", CTALinkBlock())],
+        blank=True,
+        max_num=1,
+        verbose_name="Experience CTA Link",
+    )
+    experience_content = StreamField(
+        [
+            (
+                "social_embeds",
+                StreamBlock(
+                    [("embed", EmbedBlock())], max_num=12, label="Social Embeds"
```

Comment:
Sorry, I missed one while looking at the designs now. I've updated the AC. We need also need a caption field and where the embed is from.

We can grab where the embed is from based on the value in `EmbedBlock` but that might not perform too well as we'll have to query the `Embed` model for each embed. Might be worth trying to do it though (if there's time).

Reply:
I've added this as a field the editor would need to fill in. If I understood correctly the embed source won't be known until the block is added so this approach above wouldn't help (I think). https://github.com/torchbox/rca-wagtail-2019/pull/1094/commits/5d3c097d023c4b3c9903b996af6de707b560c15e